home_work_18/index.js

```diff
@@ -0,0 +1,201 @@
+const API_KEY = "d3890f0";
+const BASE_URL = "http://www.omdbapi.com/";
+const inputContainer = document.querySelector("#input-container");
+const searchResultArea = document.querySelector("#films");
+const filmDetails = document.querySelector("#film-details");
+const pagesContainer = document.querySelector("#pages");
+const btnSearch = document.querySelector("#search");
+const inputFilmTitle = document.querySelector("#input-film-title");
+const filmsWrapper = document.querySelector("#films-wrapper");
+const statusMassage = document.createElement("span");
+const nextPage = document.createElement("button");
+
+nextPage.id = "next";
+const prevPage = document.createElement("button");
+prevPage.id = "prev";
+let currentPage = 1;
+
+pagesContainer.classList.add("pages-container");
+
+statusMassage.classList.add("status-massage");
+filmsWrapper.prepend(statusMassage);
+
+inputContainer.addEventListener("keydown", (e) => {
+  if (e.code === "Enter") {
+    getSearchResult();
+  }
+});
+btnSearch.addEventListener("click", getSearchResult);
+
+const renderFilmListToHTML = (result, pageId) => {
+  const {
+    Response: responseStatus,
+    Error: errorStatus,
+    Search: searshResult,
+    totalResults,
+  } = result;
+  if (responseStatus === "False") {
+    statusMassage.innerHTML = "";
+    searchResultArea.innerHTML = "";
+    statusMassage.classList.add("error");
+    statusMassage.innerHTML = `⚠ ${errorStatus}`;
+    pagesContainer.innerHTML = "";
+    currentPage = 1;
+  } else {
+    let arrayFilms = searshResult;
+    statusMassage.innerHTML = "";
+    statusMassage.classList.remove("error");
+    statusMassage.innerHTML = `${totalResults} results found for "${inputFilmTitle.value}"`;
+    const totalPages = Math.ceil(totalResults / 10);
+
+    if (String(totalPages?.length) && !pageId) {
+      renderPages(totalPages);
+    }
+
+    searchResultArea.innerHTML = "";
+    arrayFilms.forEach((el) => {
+      const { Poster: poster, Title: title, imdbID: moveId } = el;
+      const filmData = searchResultArea.appendChild(
+        document.createElement("div")
+      );
+      filmData.classList.add("film-style");
+      const filmItem = filmData.appendChild(document.createElement("div"));
+      filmItem.classList.add("film-item");
+      const filmPoster = filmItem.appendChild(document.createElement("img"));
+      filmPoster.classList.add("image");
+      const filmTitle = filmItem.appendChild(document.createElement("span"));
+      filmTitle.classList.add("film-title");
+
+      const buttonDetails = filmData.appendChild(
+        document.createElement("button")
+      );
+      filmPoster.src = poster;
```

Comment:
You should have a fallback poster to show, to avoid errors with movies w/o poster

<img width="621" alt="image" src="https://user-images.githubusercontent.com/84010501/204878558-68dc74b5-ca74-4bd7-8726-8deb16774376.png">